Source/Core/Core/HW/SI_GCAdapter.cpp

```diff
@@ -398,6 +399,26 @@ void Input(int chan, GCPadStatus* pad)
 	}
 }
 
+void ResetRumble()
+{
+	if (!SConfig::GetInstance().m_GameCubeAdapter)
+		return;
+	if (s_handle == nullptr || !s_detected)
+		return;
+
+	s_controller_rumble[0] = 0;
+	s_controller_rumble[1] = 0;
+	s_controller_rumble[2] = 0;
+	s_controller_rumble[3] = 0;
+
+	unsigned char rumble[5] = { 0x11, s_controller_rumble[0], s_controller_rumble[1], s_controller_rumble[2], s_controller_rumble[3] };
```

Comment:
I didn't suggest that change because it indicates better where the values come from. +/-0 either way tho.